vpp-manager/uplink/common.go

```diff
@@ -167,11 +168,16 @@ func SupportedUplinkDrivers(params *config.VppManagerParams, conf *config.Interf
 	if d := NewAFPacketDriver(params, conf); d.IsSupported(false /* warn */) {
 		lst = append(lst, d)
 	}
+	if d := NewRDMADriver(params, conf); d.IsSupported(false /* warn */) {
```

Comment:
The order is important in this function, drivers that we put first in the list will be tried first. Since RDMA is a native driver, it will be the most efficient option to drive an interface it supports, so we want to put it near the top of the list (next to virtio and avf).